<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileScanClient</class>
 <widget class="QWidget" name="TileScanClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tile Scan Client for MESc</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="lblScanner">
       <property name="text">
        <string>Scanner</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbScanner">
       <item>
        <property name="text">
         <string>resonant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>galvo</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblMode">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbMode">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Right &amp; Down</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left &amp; Down</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Right &amp; Up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left &amp; Up</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblXDim">
       <property name="text">
        <string>x Dimension</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spnDimX">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblYDim">
       <property name="text">
        <string>y Dimension</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spnDimY">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblResX">
       <property name="text">
        <string>Frame width</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="spnResolution">
       <property name="suffix">
        <string> px</string>
       </property>
       <property name="minimum">
        <number>64</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>512</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblViewPortX">
       <property name="text">
        <string>Viewport X</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spnViewportX">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblViewportY">
       <property name="text">
        <string>Viewport Y</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spnViewportY">
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblOverlap">
       <property name="text">
        <string>Overlap</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="spnOverlap">
       <property name="suffix">
        <string/>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>First X (absolute)</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QDoubleSpinBox" name="dsbFirstX">
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSlowX">
         <property name="text">
          <string>Current</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>First Y (absolute)</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QDoubleSpinBox" name="dsbFirstY">
         <property name="minimum">
          <double>-999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSlowY">
         <property name="text">
          <string>Current</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Output Directory</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditOutput"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowse">
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="17" column="1">
      <widget class="QPushButton" name="btnAbort">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QCheckBox" name="stitchTiff">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Craete Fused Tiff</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblgraphic">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/2</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
